src/30_Advanced/Linked_Dropdowns.html

```diff
@@ -0,0 +1,95 @@
+<!---{
+  "preview": "default"
+  }--->
+<!--
+  #### Introduction
+
+  These linked dropdowns are implemented using AMP components.
+  This pattern is useful when the values in the second dropdown
+  depend on the value the user selects in the first dropdown.
+  Here we select cities in a chosen country.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/advanced/linked_dropdowns/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+  <!-- #### Setup -->
+  <!--
+    Import the `amp-bind`, `amp-list` and `amp-mustache` components.
+  -->
+  <div>
```

Comment:
do we need this `div`?

Reply:
Hmm, I added it so that the 3 scripts would appear together in the example:
<img width="745" alt="screen shot 2017-10-26 at 1 49 33 pm" src="https://user-images.githubusercontent.com/2363700/32076378-9a4c0ab2-ba54-11e7-8845-04fe8e76a6bd.png">

Without the `<div>`, only the first script tag is shown, instead of all 3. Is there another way to group these? Or do you think it would be better to show the whole `<head>` tag instead of adding these invalid `<div>` tags?

Reply:
I checked and this causes the build to fail, so I think I'll show the full `<head>` instead.